test/frontend_test.dart

```diff
@@ -0,0 +1,457 @@
+//@Skip()
+@Timeout(const Duration(seconds: 45))
+import 'angular_utils.dart';
+import 'testing_utils.dart';
+import 'dart:io';
+import 'package:webdriver/io.dart';
+import 'package:test/test.dart';
+
+void main() {
+  WebDriver driver;
+  const String url = "https://www.gyerunkanyukam.hu/";
+  var newUserEmail = "connor.stansfield.development@gmail.com";
+  var oldUserEmail = "connor_stansfield@live.com";
+  var allUserPassword = "password123%";
+
+  group("http client |", () {
```

Comment:
I'm not sure if this qualifies as an E2E test. E2E tests should run in the browser by definition. It is perfectly ok and a good idea to check if the main page loads, but should be done through Selenium + webdriver, not `HttpClient`.